gateway/hail.nginx.conf.in

```diff
@@ -158,3 +161,113 @@ server {
     include /etc/letsencrypt/options-ssl-nginx.conf;
     ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
 }
+
+server {
+    server_name app.@domain@;
+
+    gzip on;
+    gzip_types  text/plain text/css text/html text/javascript application/javascript application/x-javascript text/js text/xml  application/xml application/json application/rss+xml image/svg+xml font/opentype;
+
+    location / {
+        proxy_pass http://web/;
+    }
+
+    listen [::]:443 ssl;
+    listen 443 ssl;
+    ssl_certificate /etc/letsencrypt/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/privkey.pem;
+    include /etc/letsencrypt/options-ssl-nginx.conf;
+    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
+}
+
+server {
+    server_name notebook-api.@domain@;
+
+    # Support large URI's stemming from access tokens
+    client_header_buffer_size 64k;
+    large_client_header_buffers 4 64k;
+
+    gzip on;
+    gzip_types  text/plain application/json;
+
+    location = /auth {
+        internal;
+
+        resolver kube-dns.kube-system.svc.cluster.local;
+        proxy_pass http://auth-gateway.default.svc.cluster.local/verify/$auth_request_uri;
+    }
+
+    location = /auth-notebook {
```

Comment:
Is this used?  The auth_request below is commented out.

Reply:
This will be used, placeholder until I re-enable it. It works as provided, just not on subsequent requests issued by Jupyter, which drop the ?access_token. I can remove if you prefer.